packages/@aws-cdk/aws-ecr-assets/test/image-asset.test.ts

```diff
@@ -5,27 +5,19 @@ import * as iam from '@aws-cdk/aws-iam';
 import * as cxschema from '@aws-cdk/cloud-assembly-schema';
 import { App, DefaultStackSynthesizer, IgnoreMode, Lazy, LegacyStackSynthesizer, Stack, Stage } from '@aws-cdk/core';
 import * as cxapi from '@aws-cdk/cx-api';
+import { testFutureBehavior } from 'cdk-build-tools/lib/feature-flag';
 import { DockerImageAsset } from '../lib';
 
 /* eslint-disable quote-props */
 
 const DEMO_IMAGE_ASSET_HASH = 'b2c69bfbfe983b634456574587443159b3b7258849856a118ad3d2772238f1a5';
 
-
-let app: App;
-let stack: Stack;
-beforeEach(() => {
-  app = new App({
-    context: {
-      '@aws-cdk/aws-ecr-assets:dockerIgnoreSupport': true,
-    },
-  });
-  stack = new Stack(app, 'Stack');
-});
+const flags = { '@aws-cdk/aws-ecr-assets:dockerIgnoreSupport': true };
 
 describe('image asset', () => {
-  test('test instantiating Asset Image', () => {
+  testFutureBehavior('test instantiating Asset Image', flags, App, (app) => {
```

Comment:
I really want a `describeFutureBehavior` and/or `beforeEachWithFutureBehavior` so we can just designate a whole block at a time...

Reply:
I've thought of a `describeFutureBehavior` too, but involves a bit more effort. `beforeEachWithFutureBehavior` might be easier though.

I don't think we'll have more cases to deal with 🤞 so I'deferring this to later if we need it.